SPECS/cal10n/cal10n.spec

```diff
@@ -73,58 +74,54 @@ http://cal10n.qos.ch/manual.html
 EOF
 
 %build
-for dir in cal10n-api
-do
-  pushd $dir
-  export CLASSPATH=$(build-classpath \
-                     junit \
-                     ):target/classes:target/test-classes
-  ant -Dmaven.mode.offline=true package javadoc \
-      -Dmaven.test.skip=true \
-      -lib %{_datadir}/java
-  popd
-done
+mkdir -p lib
+build-jar-repository -s lib \
+%if %{with tests}
+    ant-antunit \
+%endif
+    ant/ant
+%{ant} \
+%if %{without tests}
+    -Dtest.skip=true \
+%endif
```

Comment:
We don't use `with tests` but the `with_check` macro. Please update the spec - right now we'd never run the `with test` branch.

Reply:
Got it. Updated.